pom.xml

```diff
@@ -523,6 +524,22 @@
             <version>1.0-1</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
```

Comment:
What's this for? can we remove it?

Reply:
yes it is... these are the brokers that enable spring kafka to use 2.8.1...more detail can be found here: https://giters.com/spring-projects/spring-kafka/issues/1792

i tested it to make sure that it works correctly and is required